registry/txt_test.go

```diff
@@ -216,13 +216,13 @@ func testTXTRegistryRecordsPrefixed(t *testing.T) {
 	r, _ := NewTXTRegistry(p, "txt.", "", "owner", time.Hour, "wc", []string{}, false, nil)
 	records, _ := r.Records(ctx)
 
-	assert.True(t, testutils.SameEndpoints(records, expectedRecords))
+	assert.ElementsMatch(t, expectedRecords, records)
```

Comment:
Unfortunately, `SameEndpoints` is needed to allow for differences in representation that have no semantic difference in meaning. For example, the ordering of the Targets and ProviderSpecific slices is not semantically meaningful. So `ElementsMatch` is too strict.